src/pages/Login/index.jsx

```diff
@@ -1,3 +1,183 @@
+"use client";
+import React from "react";
+import { useNavigate } from "react-router-dom";
+
+//for field validation
+import { object, string } from "yup";
+
+//nextUI components
+import { Image, Input, Button } from "@nextui-org/react";
+import { EyeFilledIcon } from "./EyeFilledIcon";
+import { EyeSlashFilledIcon } from "./EyeSlashFilledIcon";
+
+import authBackground from "../../assets/authBackground.png";
+
+let userSchema = object({
+  email: string()
+    .required("Username is required")
+    .matches(
+      /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,6}$/,
+      "Invalid Email Format"
+    )
+    .label("email"),
+  password: string().required("Password Is Required").label("password"),
+});
+
 export const LoginPage = () => {
-  return <p>Job Quest Login</p>;
+  const [userInfo, setUserInfo] = React.useState({
+    email: "",
+    password: "",
+  });
+
+  //used to shwo or not show password.
+  const [isVisible, setIsVisible] = React.useState(false);
+
+  //form loading
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [errors, setErrors] = React.useState({});
+
+  //Router navigation
+  const navigateTo = useNavigate();
+
+  React.useEffect(() => {
+    if (userInfo.email === "" || userInfo.password === "") {
+      setErrors((prevErrors) => ({
+        ...prevErrors,
+        email: "",
+        password: "",
+      }));
+    }
+  }, [userInfo]);
+
+  async function handleSubmit() {
```

Comment:
Overall, the `handleSubmit` function has a good approach to simulating the email/password validation. 
A limitation to note though, is that it will only work if every user that accesses the website always creates an account first (which stores the details in localStorage) before they login

A quick fix would be to also cater for that use case (this would be in my suggested update)